packages/logo/src/docs/Component.stories.mdx

```diff
@@ -0,0 +1,100 @@
+import { Meta, Story, Props } from '@storybook/addon-docs';
+import { ComponentHeader, TabContainer } from 'storybook/blocks';
+
+import Logo from '../index';
+import { version } from '../../package.json';
+import Description from './Description.mdx';
+import Changelog from '../../CHANGELOG.md';
+
+<!-- Meta -->
+
+<Meta title="Components/Logo" component={Logo} argTypes={{}} />
+
+export const LogoTemplate = args => <Logo {...args} />;
+
+<!-- Canvas -->
+
+<Story
+  name="Logo"
+  args={{
+    color: 'primary',
+    fontSize: 200,
+  }}
+  parameters={{
+    backgrounds: {
+      values: [
+        { name: 'primary', value: '#052962' },
+        { name: 'secondary', value: '#FFFFFF' },
+        { name: 'tertiarys', value: '#F1F4FA' },
```

Comment:
do not need to replace color for the whole canvas. Could you please do the similar as for the documentation part?
justthe  wrap logo to the square div, and change the background of the div depending on the chosen color of the logo.